packages/palette_generator/lib/palette_generator.dart

```diff
@@ -188,6 +235,7 @@ class PaletteGenerator with Diagnosticable {
       avoidRedBlackWhitePaletteFilter
     ],
     List<PaletteTarget> targets = const <PaletteTarget>[],
+    bool useIsolate = true,
```

Comment:
How much overhead is there in spawning an isolate? Is this actually desirable behavior as a default (e.g., will this have worse performance when run on a small image)?

Reply:
I'll do some proper testing to see, but in my experience, using an isolate for palette_generator is always smoother than not doing so.

Reply:
Yes. Without isolate using .fromImageProvider() from palettegenerator lag a lot.

Totally unusable on scrollable lists